data/Lavishmunjal.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Lavish Munjal",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Computer Science Student || Learning Java + DSA || Learning Web Development || Bachelor of Technology (BTech) || Tech Enthusiast",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Lavishmunjal8",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on LinkdIn",
+      "url": "https://www.linkedin.com/in/lavishmunjal/",
+      "icon": "FaLinkdin"
```

Comment:
```suggestion
      "icon": "FaLinkedin"
```

Reply:
Hello, where i found my linkfree link??

Reply:
> Hello, where i found my linkfree link??

Your profile will be visible and added once the PR is merged. 🙂